src/state/presentationManagerImpl.test.ts

```diff
@@ -334,4 +340,23 @@ describe('presentationManager', () => {
 
     expect(await states).toEqual([{ type: 'idle' }]);
   });
+
+  it('should toggle the edit mode', () => {
+    presentationManager.toggleEditMode();
+    presentationManager.toggleEditMode();
+
+    expect(communicationChannel.broadcastMessage).toBeCalledTimes(2);
+
+    expect(communicationChannel.broadcastMessage).toHaveBeenNthCalledWith(
+      1,
+      'net.nordeck.whiteboard.present_slide',
+      { view: { isEditMode: true, slideId: 'slide-0' } }
+    );
+
+    expect(communicationChannel.broadcastMessage).toHaveBeenNthCalledWith(
+      2,
+      'net.nordeck.whiteboard.present_slide',
+      { view: { isEditMode: false, slideId: 'slide-0' } }
+    );
+  });
```

Comment:
There are more cases that are implemented but not yet tested:
* should accept the edit mode from a different user
* should deactivate the edit mode when the slide changes